core/src/commonMain/kotlin/Peripheral.kt

```diff
@@ -138,3 +140,19 @@ public interface Peripheral {
         onSubscription: OnSubscriptionAction = {},
     ): Flow<ByteArray>
 }
+
+/** Remember the bluetooth connected is not the same thing as Kable's [State.Connected]. */
```

Comment:
I think it will be easier to read if we're more explicit.

```suggestion
/**
 * Suspends until [Peripheral] receiver is connected at the bluetooth layer, but [Peripheral] not yet ready for general
 * use (i.e. services have not been discovered and observations have not been wired up).
 *
 * @see suspendUntilConnected for implementation that suspends until ready (Kable's [State.Connected]).
 */
```